airflow/cli/cli_parser.py

```diff
@@ -90,8 +93,7 @@
     dup = {k for k, v in Counter([c.name for c in airflow_commands]).items() if v > 1}
     raise CliConflictError(
         f"The following CLI {len(dup)} command(s) are defined more than once: {sorted(dup)}\n"
-        f"This can be due to the executor '{ExecutorLoader.get_default_executor_name()}' "
-        f"redefining core airflow CLI commands."
+        f"This can be due to an Executor or Auth Manager redefining core airflow CLI commands."
```

Comment:
Auth managers can also [vend](https://airflow.apache.org/docs/apache-airflow/stable/core-concepts/auth-manager.html#cli) CLI commands, so this message needs to be updated as well.